Code/Editor/CryEdit.cpp

```diff
@@ -2810,7 +2810,9 @@ void CCryEditApp::OpenProjectManager(const AZStd::string& screen)
 {
     // provide the current project path for in case we want to update the project
     AZ::IO::FixedMaxPathString projectPath = AZ::Utils::GetProjectPath();
-#if !AZ_TRAIT_OS_PLATFORM_APPLE && !AZ_TRAIT_OS_USE_WINDOWS_FILE_PATHS
+#if defined(AZ_PLATFORM_LINUX)
+    const char* argumentQuoteString = "";
+#elif !AZ_TRAIT_OS_PLATFORM_APPLE && !AZ_TRAIT_OS_USE_WINDOWS_FILE_PATHS
```

Comment:
The `!AZ_TRAIT_OS_PLATFORM_APPLE && !AZ_TRAIT_OS_USE_WINDOWS_FILE_PATHS` is for Linux.

Therefore that `argumentQuoteString` constant should be set to empty.

But this change would cause another issue.
If the ProjectManager is in a path with a space in it, wouldn't this fail?

Reply:
This change might not be needed, I was trying different things to resolve the launching issues (before discovering the issue with the environment variables), and this solution was taken from a similar fix for launch the Lua Editor (which uses Qt's launch function instead). Will try to undo this change and try again.